src/parser/priest/shadow/modules/talents/ShadowCrash.tsx

```diff
@@ -2,39 +2,47 @@ import React from 'react';
 
 import SPELLS from 'common/SPELLS/index';
 import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
-import Events, { DamageEvent } from 'parser/core/Events';
+import Events, { DamageEvent, EnergizeEvent } from 'parser/core/Events';
 import Statistic from 'interface/statistics/Statistic';
 import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
 import BoringSpellValueText from 'interface/statistics/components/BoringSpellValueText';
 
 import ItemDamageDone from 'interface/ItemDamageDone';
 import { formatNumber } from 'common/format';
 import AbilityTracker from 'parser/priest/shadow/modules/core/AbilityTracker';
+import Insanity from 'interface/icons/Insanity'
 
 // Example Log: /report/zgBQ3kr6aAv19MXq/22-Normal+Zul+-+Kill+(2:26)/3-Selur
 class ShadowCrash extends Analyzer {
   static dependencies = {
     abilityTracker: AbilityTracker,
   };
+
   damage: number = 0;
+  insanityGained: number = 0;
```

Comment:
dont need to specify that these are numbers if you are assigning a value to it immediately. it will figure that out itself.